<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WPageCtlWidgetClass</class>
 <widget class="QWidget" name="WPageCtlWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>36</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>36</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>36</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WPageCtlWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font: 12px &quot;微软雅黑&quot;;
}

QPushButton[sel=false]{
	border:none;
	background-color: #BEC8D0;
	border-radius:4px;
	background-position: center;
    background-repeat: no-repeat;
}

QPushButton[sel=false]::hover{
	border:none;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #0066FF, stop:1 #BD00FF);
	border-radius:4px;
	background-position: center;
    background-repeat: no-repeat;
}

QPushButton[sel=true]{
	border:none;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #0066FF, stop:1 #BD00FF);
	border-radius:4px;
	background-position: center;
    background-repeat: no-repeat;
}

QPushButton#pb_pre{
	background-image: url(:/QtTest/icon/Setting/pre.png);
}

QPushButton#pb_next{
	background-image: url(:/QtTest/icon/Setting/next.png);
}

QComboBox{
	border:1px solid #E0E7ED;
	border-radius:4px;
}
QComboBox::down-arrow {
	image: url(:/QtTest/icon/Speech/down.png);
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_total_size">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>17</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>17</height>
          </size>
         </property>
         <property name="text">
          <string>entries of 430</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>88</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pb_pre">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>44</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pb_1">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_2">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_about">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_last">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>9</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_next">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>44</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../AiSound.qrc"/>
 </resources>
 <connections/>
</ui>
